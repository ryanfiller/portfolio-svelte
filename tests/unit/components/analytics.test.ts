import { describe, it, expect } from 'vitest'
import { render } from '@testing-library/svelte'
import type { RenderResult } from '@testing-library/svelte'

import Analytics from '$components/analytics.svelte'

describe('<Analytics />', () => {
	let component: RenderResult<Analytics>

	it('renders the correct urls', () => {
		component = render(Analytics)
		expect(component.container).toMatchSnapshot()
	})
})
