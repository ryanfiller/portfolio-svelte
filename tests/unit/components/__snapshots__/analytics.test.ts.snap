// Vitest Snapshot v1

exports[`<Analytics /> > renders the correct urls 1`] = `
<body>
  <div>
     
    <!--&lt;Analytics&gt;-->
  </div>
</body>
`;
